services/api/service.go

```diff
@@ -1733,6 +1739,45 @@ func (api *RelayAPI) checkSubmissionSlotDetails(w http.ResponseWriter, log *logr
 		return false
 	}
 
+	// Do a check in memchache for the slot being with a registered provider. If miss, go to redis. if miss again go to postgres.
+
+	// Find the duty for the submission slot
```

Comment:
Is it intended for this logic to still run if `api.opts.MevCommitFiltering` is `false`?